ui/lib/core/addon/helpers/sanitized-html.js

```diff
@@ -0,0 +1,15 @@
+import { helper } from '@ember/component/helper';
+import { debug } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import { sanitize } from 'dompurify';
+
+export default helper(function sanitizedHtml([htmlString]) {
+  try {
+    return htmlSafe(sanitize(htmlString));
+  } catch (e) {
+    debug('Error sanitizing string', e);
+    // I couldn't get this to actually fail but as a fallback,
+    // render the value as-is with HTML escaping
+    return htmlString;
```

Comment:
Nit on the comment but wouldn't the string be returned as is _without_ HTML escaping?

Reply:
no, I struggled with this verbage so open to other ways of saying it. The [htmlSafe docs](https://api.emberjs.com/ember/release/functions/@ember%2Ftemplate/htmlSafe) state that using `htmlSafe` will mean that string "will not be HTML escaped" -- by default the template engine HTML-escapes all content.

Reply:
I like that a lot better! Words are hard 😄